tests/test_async_processing_import_export.py

```diff
@@ -32,9 +32,15 @@
 except ModuleNotFoundError:
     from test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
 
+try:
+    import actinia_stac_plugin
+    no_stac_plugin = False
+except Exception:
+    no_stac_plugin = True
```

Comment:
This was the best option that cam up to my mind on how to verify the installation. Any suggestion to make it cleaner?

Reply:
Fine for me. What do you think @anikaweinmann ?

Reply:
For me it is also fine